docs/reference/cli-commands/run.md

```diff
@@ -26,64 +26,6 @@ pebble run --args myservice --port 8080 \; --hold
           --http=        Start HTTP API listening on this address (e.g., ":4000")
       -v, --verbose      Log all output from services to stdout
           --args=        Provide additional arguments to a service
+          --identities=  Seed identities from file (like update-identities --replace)
 ```
 <!-- END AUTOMATED OUTPUT -->
-
-## How it works
```

Comment:
Was this content supposed to be deleted?

Reply:
Might be a bug in the script, let me double check.